ios/CovidShield/Info.plist

```diff
@@ -60,10 +60,10 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
-  <!-- uncomment the next 4 lines to use the QR feature -->
-	<!-- <key>NSCameraUsageDescription</key>
+	<key>NSCameraUsageDescription</key>
 	<string>Allow $(PRODUCT_NAME) to use the camera</string>
-	<key>NSMicrophoneUsageDescription</key>
+  <!-- uncomment the next 2 lines to use the QR feature -->
+	<!-- <key>NSMicrophoneUsageDescription</key>
 	<string>Allow $(PRODUCT_NAME) to use the microphone</string> -->
```

Comment:
This line should be commented out too, or line 66 should be uncommented.

Reply:
Updated